### PR TITLE
Support for RHEL and CentOS platforms

### PR DESCRIPTION
**Pull request by danielwhite**
This is a preliminary pull request, just to get some early feedback around the approach I've taken to this.

Far more testing is also required before I'd be satisfied with this.

### CODE REVIEW

**Comment by UnderGreen** [comment]
Thank you for this PR. I will try to fix testing today or tomorrow.

**Comment by danielwhite** [comment]
@UnderGreen, I've rebased against the fix in #62 to address the build failure.

**Comment by UnderGreen** [comment]
https://travis-ci.org/UnderGreen/ansible-role-mongodb/builds/128535620
There problem fixed. Please rebase from master and drop b77b57b and 99656df commits from this PR.

File: tasks/main.yml [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
+  with_first_found:
+    - "{{ ansible_distribution }}.yml"
+    - "{{ ansible_os_family }}.yml"
+
 - name: Include installation on Debian-based OS
   include: install.deb.yml
   when: ansible_os_family == 'Debian'
   tags: [mongodb]
 
+- name: Include installation on RedHat-based OS
```
Comment (UnderGreen) [inline tasks/main.yml]: It's possible to combine with previous task:
include: "install.{{ ansible_os_family | lower }}.yml"

**Comment by UnderGreen** [comment]
I know how to fix Ubuntu 12.04 build. Just give me some time. Need a build docker image with full python version.

http://bertvv.github.io/notes-to-self/2015/12/13/testing-ansible-roles-with-travis-ci-part-2-multi-platform-tests/ - maybe this article will help you with centos.

**Comment by danielwhite** [comment]
Thanks, I'll play with the CentOS side a bit more locally. I've found Travis to be different enough in the past that I tend to throw changes at it first to see what breaks before diving deeper.

**Comment by UnderGreen** [comment]
@danielwhite please rebase from master, I fixed tests for 12.04 Ubuntu test suite.

**Comment by danielwhite** [comment]
Cheers. I'll sort out the tests for this.

**Pull request merged**